tests/ptf_runner.py

```diff
@@ -111,6 +111,15 @@ def ptf_runner(host, testdir, testname, platform_dir=None, params={},
         host.create_macsec_info()
 
     try:
+        if pdb:
+            # Write command to file. Use short test name for simpler launch in ptf container.
+            script_name = "/tmp/" + testname.split(".")[-1] + ".sh"
+            with open(script_name, 'w') as f:
```

Comment:
Just question, the 'with' block will close the file even if there is expection?

Reply:
Yes, if an exception is raised in the `with` statement the file will still close.